manifests/ResponsivelyApp/ResponsivelyApp/0.1.5.yaml

```
@@ -0,0 +1,17 @@
+Id: ResponsivelyApp.ResponsivelyApp
+Version: 0.1.5
+Name: ResponsivelyApp
+Publisher: ResponsivelyApp
+License: © 2020 ResponsivelyApp, Inc. All rights reserved.
+AppMoniker: ResponsivelyApp
+Tags: responsively,app,viewpoint
+Description: A modified browser built using Electron that helps in responsive web development.
+Homepage: https://manojvivek.github.io/responsively-app/
+Installers:
+  - Arch: x64
+    Url: https://github.com/manojVivek/responsively-app/releases/download/v0.1.5/ResponsivelyApp-Setup-0.1.5.exe
+    Sha256: b359f668eee2f28847b9fc5acc211877509adaa0f6e663b24f0a659b1c2a6686
```

```suggestion
    Sha256: F27E6E72E2A921AF897DF6646153FB3AF0808D7348BFF96F97E0D94A1FDEC88C
```

This should be the correct hash for the 0.1.5 installer, according to `Get-FileHash`.

---

Yes, I just changed. Thanks for your inputs :)